_std/macros/_macros.dm

```diff
@@ -148,3 +148,5 @@ while(FALSE)
 
 var/global/list/addr_padding = list("00000", "0000", "000", "00", "0", "")
 #define BUILD_ADDR(TYPE_ID, NUM) "\[0x[TYPE_ID][addr_padding[length(num2text(NUM, 0, 16))]][num2text(NUM, 0, 16)]\]"
+
+#define percentmult(perc, mult) (100-((100-clamp((perc), 0, 100))**mult))
```

Comment:
is raising this to the mult power intentional?

Reply:
Intentional, so it is more equivalent to running the prob multiple times instead of simple multiplication of the probability:
> (50% chance with mult 2 should be 75% chance, not 100%)